platforms/mobile/ios.md

```diff
@@ -1 +1,13 @@
-# Langley Foxall iOS Apps
\ No newline at end of file
+# Langley Foxall iOS
+
+iOS is a big part of the company and it's what got us started. We write our iOS apps in purely Swift.
+
+Because Swift is a reletively new language and that iOS itself changes every year, ensuring we have an up to date guide full of good-to-knows & best practices is the key to ensuring we stay on top of iOS.
+
+## Good to Knows
+- To submit your app to the app store you have to upload iPad Pro screenshots, this can cause issues because we often don't design our apps for the 12.9" display and because of this the constraints for the app's layout could not be compatible. Please ensure your app is compatible with the iPad Pro at a minimum.
```

Comment:
It's worth specifying this is for (I think) iPad only apps. iPhone apps need iPhone X screenshots and Regular ones.